platform/android/MapboxGLAndroidSDKTestApp/src/main/java/com/mapbox/mapboxsdk/testapp/activity/maplayout/MapInDialogActivity.java

```diff
@@ -72,6 +72,8 @@ public Dialog onCreateDialog(Bundle savedInstanceState) {
         @Override
         public void dismiss() {
           if (mapView != null && !destroyed) {
```

Comment:
with introduction of https://github.com/mapbox/mapbox-gl-native/pull/12373 management of this destroyed flag is obsolete and can be replaced with `mapView.isDestroyed()` instead.